### ISSUE TITLE
I am trying to use this puppet module, but errors arise

### ISSUE THREAD

**Issue by novalore** [javiyt/puppet-ispconfig#8]
Hi,
That is what happens on my virtual server when using this ispconfig module:

``` bash
root@aea:~# puppet agent --server gandi19 --test
err: Could not retrieve catalog from remote server: Error 400 on SERVER: Invalid parameter config_hash at /etc/puppet/environments/production/modules/ispconfig/manifests/mail.pp:15 on node aea
warning: Not using cache on failed catalog
err: Could not retrieve catalog; skipping run
```

That error appears only once I comment out line 50 of init.pp which is giving me the same hard time I see in another issue.
